packages/std/src/query/wasm.rs

```diff
@@ -47,23 +47,14 @@ pub struct ContractInfoResponse {
 
 impl QueryResponseType for ContractInfoResponse {}
 
-impl ContractInfoResponse {
-    /// Constructor for testing frameworks such as cw-multi-test.
-    /// This is required because query response types should be #[non_exhaustive].
-    /// As a contract developer you should not need this constructor since
-    /// query responses are constructed for you via deserialization.
-    #[doc(hidden)]
-    #[deprecated(
-        note = "Use ContractInfoResponse::default() and mutate the fields you want to set."
-    )]
-    pub fn new(code_id: u64, creator: impl Into<String>) -> Self {
-        ContractInfoResponse {
-            code_id,
-            creator: creator.into(),
-            ..Default::default()
-        }
-    }
-}
+impl_response_constructor!(
+    ContractInfoResponse,
+    code_id: u64,
+    creator: String,
+    admin: Option<String>,
+    pinned: bool,
+    ibc_port: Option<String>
+);
```

Comment:
With this solution, let's also remove the Default implementation of ContractInfoResponse. There is no point in creating an instance with e.g. code_id: 0.

Then if we don't need Default anymore, we can make both creator and admin: String -> Addr as we discussed in other places.

Reply:
I already did both in #1883

Reply:
Great, thanks